### DISCUSSION TITLE
Acquire a dedicated server for our group

### DISCUSSION THREAD

**Discussion by rawaa123** [ude-soco/CourseMapper-webserver#1261] [Q&A]
Hello @ralf-berger,
If we want to acquire a dedicated server for our group that can run all our projects, can this be arranged before the end of the year to complete all procedures related to the payment for the new server? If yes, could you please provide the steps and requirements? Also, will it be a better solution than the current one we are using, in terms of resources specifications and cost?

**Comment by ralf-berger** [comment]
Unfortunately I don't see a way to reliably operate our own hardware on site, since connectivity is subject to arbitrary ZIM regulations and processes. We can rent a dedicated machine though, resulting in more performance for the same money, but at the cost of flexibility. Payments would still be on a monthly basis though. So far I haven't found any provider that offers paying a larger sum upfront.

**Reply by ralf-berger** [reply]
I called Hetzner to explicitly ask if there are any options to buy or pre-pay a server for more than a month and they told me that unfortunately it's not possible. Apparently you can transfer credit to your customer account in advance, but servers are still billed on a monthly basis.
